s3/pkg/datastore/yig/conf/yig.toml

```diff
@@ -9,4 +9,5 @@ url="default"
 ceph_dir="/etc/ceph/*.conf"
 
 [database]
+db_type="tidb"
 db_url="root:@tcp(tidb:4000)/yig"
```

Comment:
it is highly unsafe to have a root user without a password and use it in application. please do
1. run the mysql secure install script (hardening the mysql)
2. create an application user on the DB, with priveleges granted to what is needed by the application
3. use that user and credentials in our connection code

Reply:
@rhsakarpos thanks for your kindly review. and the yig.toml in our source code is just the sample. and it is only used in ci/test environment. In production environment, the deployment team will handle this problem. and i have link this problem into 'https://github.com/opensds/multi-cloud/issues/683', and add comment in that issue. thanks a lot.